BREAKING.md

```diff
@@ -189,6 +190,10 @@ Additionally, please note that the `Connecting` state is being renamed to `Catch
 `ConnectionState.Connecting` is marked as deprecated, please use `ConnectionState.CatchingUp` instead.
 `ConnectionState.Connecting` will be removed in the following major release.
 
+### Remove ICodeLoader from container-definitions
+`ICodeLoader` in `@fluidframework/container-definitions` was deprecated a while ago and is now removed. Use `ICodeDetailsLoader` by importing it from `@fluidframework/container-loader`.
+Please refer to [replace ICodeLoader with ICodeDetailsLoader interface](#Replace-ICodeLoader-with-ICodeDetailsLoader-interface) for more details.
```

Comment:
I think it's probably better to just copy the details from the section you referenced so people don't have to go searching for it